behavior/MouseTracking/get_mouse_XY_pos_live_monitor.m

```diff
@@ -0,0 +1,112 @@
+function [ centroids ] = get_mouse_XY_pos_live_monitor( movie )
```

Comment:
Generally speaking, one should not duplicate functions like `get_mouse_XY_pos` and `get_mouse_XY_pos_live_monitor`. Rather, `get_mouse_XY_pos` should have a user-specified flag that indicates the output intermediate results or not.